framework/include/parser/Syntax.h

```diff
@@ -207,6 +211,10 @@ class Syntax
   /// The list of Moose system objects to tasks.  This map indicates which tasks are allowed to build certain MooseObjects.
   std::multimap<std::string, std::string> _moose_systems_to_tasks;
 
+  /// The list of deprecated Moose system objects to tasks. If an item is in this map, the task is still allowed to build
+  /// the object from the MOOSE system, but it is deprecated to do so.
+  std::multimap<std::string, std::string> _deprecated_moose_systems_to_tasks;
```

Comment:
It would be good to try and clarify this member more. The first sentence of the doxygen and the name of the member suggest that the system itself is deprecated. But I believe you are only using this for adding `FuctorMaterials` under `add_material`; `FunctorMaterials` themselves are not a deprecated system, but adding them under `Materials` syntax with the `add_material` task is (if I understand correctly)

Reply:
yeah it s deprecated ( moose systems to tasks ).
instead of moose_depcreated_systems_to_tasks.

Reply:
renamed and change docstring